Makefile.doc

```diff
@@ -301,9 +301,12 @@ doc/tools/docgram/fullGrammar: $(DOC_GRAM) $(DOC_MLGS)
 	$(HIDE)$(DOC_GRAM) -short -no-warn $(DOC_MLGS)
 
 #todo: add a dependency of sphinx on updated_rsts when we're ready
-doc/tools/docgram/orderedGrammar doc/tools/docgram/updated_rsts: doc/tools/docgram/fullGrammar $(DOC_GRAM) $(DOC_EDIT_MLGS)
+doc/tools/docgram/orderedGrammar doc/tools/docgram/updated_rsts: #doc/tools/docgram/fullGrammar $(DOC_GRAM) $(DOC_EDIT_MLGS)
```

Comment:
Keeping the removed items around as a comment?

Reply:
this an the `false` were testing remains, now removed